verification/curator-service/src/controllers/sources.ts

```diff
@@ -0,0 +1,41 @@
+import { Request, Response } from 'express';
+
+/**
+ * List the sources.
+ */
+export const list = (req: Request, res: Response) => {
+    res.sendStatus(501);
+    return;
+};
+
+/**
+ * Get a single source.
+ */
+export const get = (req: Request, res: Response) => {
+    res.sendStatus(501);
+    return;
+};
+
+/**
+ * Update a single source.
+ */
+export const update = (req: Request, res: Response) => {
+    res.sendStatus(501);
+    return;
+};
+
+/**
+ * Create a single source.
+ */
+export const create = (req: Request, res: Response) => {
+    res.sendStatus(501);
+    return;
+};
+
+/**
+ * Delete a single source.
+ */
+export const del = (req: Request, res: Response) => {
```

Comment:
I like the naming convention for controllers (just the method name, since we're importing the entire module). but hyper-nit here: can we just spell out delete (it's the same length as update)?

Reply:
No:
'delete' cannot be called on an identifier in strict mode.ts(1102)
it's an actual keyword :(